frontend/src/components/elements/CheckInput.js

```diff
@@ -61,7 +61,7 @@ const CheckInput = ({
   type = "checkbox",
   name,
   onChange,
-  checked,
+  checked = null,
```

Comment:
I would say this default value should be `false` instead of `null`. `checked` implies a boolean. (it's CHECKED or not)